diff --git a/demos/code/code.js b/demos/code/code.js
index ed1d520..6164c4d 100644
--- a/demos/code/code.js
+++ b/demos/code/code.js
@@ -238,7 +238,7 @@ Code.LANG = Code.getLang();
  * List of tab names.
  * @private
  */
-Code.TABS_ = ['blocks', 'javascript', 'php', 'python', 'dart', 'lua', 'xml'];
+Code.TABS_ = ['blocks', 'javascript', '$languageName', 'php', 'python', 'dart', 'lua', 'xml'];
 
 Code.selected = 'blocks';
 
@@ -305,6 +305,8 @@ Code.renderContent = function() {
     xmlTextarea.focus();
   } else if (content.id == 'content_javascript') {
     Code.attemptCodeGeneration(Blockly.JavaScript, 'js');
+  } else if (content.id == 'content_$languageName') {
+    Code.attemptCodeGeneration(Blockly.$languageName, '$languageName');
   } else if (content.id == 'content_python') {
     Code.attemptCodeGeneration(Blockly.Python, 'py');
   } else if (content.id == 'content_php') {
